Readme testing update (#4787)
bbc/simorgh: README.md

```

It will spin up a production server on port 7080 and run the Cypress tests against that.
To run the smoke tests interactively, run:

```
npm run test:e2e:interactive
```

This loads a user interface which easily allows for individual tests to be run alongside a visual stream of the browser, as the tests run.

#### Environment variables

There are several environment variables you can use with our test suite, which are:

| Envionment variable  | Effect                                                                                                        | Possible values                                   |
| -------------------- | ------------------------------------------------------------------------------------------------------------- | ------------------------------------------------- |
| CYPRESS_ONLY_SERVICE | Restricts to running only the specified service                                                               | A single service i.e. `CYPRESS_ONLY_SERVICE=urdu` |
| CYPRESS_APP_ENV      | Runs the tests in a specific environment                                                                      | `test`, `local`, `live`                           |

> | CYPRESS_SMOKE        | Runs only smoke tests if true                                                                                 | boolean                                           |

Reviewer: ```suggestion
| CYPRESS_SMOKE        | Runs only smoke tests if true                                                                                 | `true`, `false`                                          |
```

> | CYPRESS_UK           | See [running e2es in the UK against Live](https://github.com/bbc/simorgh/#running-e2e-in-the-uk-against-live) | boolean                                           |

Reviewer: ```suggestion
| CYPRESS_UK           | See [running e2es in the UK against Live](https://github.com/bbc/simorgh/#running-e2e-in-the-uk-against-live) | `true`, `false`                                           |
```

> | CYPRESS_SKIP_EU      | See [running e2es outside EU](https://github.com/bbc/simorgh/#running-e2e-outside-eu)                         | boolean                                           |

Reviewer: ```suggestion
| CYPRESS_SKIP_EU      | See [running e2es outside EU](https://github.com/bbc/simorgh/#running-e2e-outside-eu)                         | `true`, `false`                                           |
```

These commands can be run in combination.

> #### Smoke tests

Reviewer 2: I would rename this full suite of tests or something similar, as we are already stating above that `npm run test:e2e` only runs the smoke tests.
